### PR TITLE
Remove eg dependency#352

### PR DESCRIPTION
**Pull request by sculove**
## Issue
#352, #324 

## Details
- remove eg dependency
- change from 'jQuery' to '$'

## Preferred reviewers
@happyhj

### CODE REVIEW

File: src/plugin/persist.js [hunk 1]
```diff
@@ -4,13 +4,18 @@
...
 // jscs:disable maximumLineLength
-eg.module("persist", ["jQuery", eg, window, document], function($, ns, global, doc) {
+eg.module("persist", ["jQuery", window, document], function($, global, doc) {
 	"use strict";
 
 	// jscs:enable maximumLineLength
 	var wp = global.performance;
 	var history = global.history;
-	var userAgent = global.navigator.userAgent;
+	var isNeeded = (function() {
```
Comment (happyhj) [inline src/plugin/persist.js]: Please add comment describes the regular expression. like device range..

Comment (sculove) [inline src/plugin/persist.js]: ok

**Comment by happyhj** [comment]
LGTM

**Comment by mixed** [comment]
We have decided that each component did select inner / outer lib of dependency.

**Pull request merged**